### PR TITLE
Customize feedstock for TileDB-Inc

### PR DESCRIPTION
**Pull request by jdblischak**
This is a quality of life PR. I noticed the badges and instructions in the README were all for the conda-forge org and channel. Now they apply to TileDB-Inc GitHub org and the tiledb channel.

Specifically, I did the following:

* `user_or_org: TileDB-Inc` updates some URLs in the README
* `tooling_branch_name: main` this ensures that future conda-smithy rerenders don't undo the changes from PR #58
* `channels` updates the badge links to anaconda.org as well as the README installation instructions (note that the channels set in `recipe/conda_build_config.yaml` affect the build process, but not the feedstock rerendering)

### CODE REVIEW

**Comment by jdblischak** [comment]
More details at https://conda-forge.org/docs/maintainer/conda_forge_yml.html

**Review by ihnorton — APPROVED** [review]
LGTM after changing the gcc target back to 10 as discussed.

**Comment by jdblischak** [comment]
@ihnorton Updated so it only rerenders the README for now

Strangely, the CI still wasn't triggered. Also, the CI jobs for PR #42 weren't restarted after you opened and closed the PR this morning

The azure jobs have been running as recently as this Tuesday, so I'm not sure what happened to change this

https://dev.azure.com/TileDB-Inc/CI/_build?definitionId=35

**Comment by jdblischak** [comment]
I can now see CI jobs displayed at the bottom of the PR (screenshot below), but they aren't functional. They can't be clicked, the Checks tab for the PR is still empty, and these jobs don't exist on [Azure](https://dev.azure.com/TileDB-Inc/CI/_build?definitionId=35&_a=summary)

image

I'll try opening and closing again, but I'm at a loss at how to fix this

**Comment by jdblischak** [comment]
@awenocur Thanks for investigating. I'm changed this PR to a draft for now. I'll return to it once we've merged #62

**Comment by jdblischak** [comment]
Two updates post commit a5c01af09e591a05cdab5712f1fea2e4ad164d6d:

1. The rerendering in that commit reverted the tooling branch back to "master", effectively undoing PR #58. Defining `tooling_branch_name: main` will prevent this from happening again in the future
2. Now that htslib is being built from source due to solver issues, I removed the bioconda channel

@awenocur Could you please review and merge? I can't get the CI jobs to run no matter what I do. But as you can see, the edits I made don't affect the recipe build process, so I don't think the tests are necessary.

**Review by Shelnutt2 — APPROVED** [review]

**Pull request merged**
